sig-auth/annual-report-2021.md

```diff
@@ -0,0 +1,117 @@
+
+- Yes. Our chairs, leads, contributors, participants, and subproject owners are from various companies.
+
+## Current initiatives and project health
+
+- Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members. 
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+- [CSR v1](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1513-certificate-signing-request)
+- [Token Request / bound SA token admission](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1205-bound-service-account-tokens)
+- [client-go auth plugins](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/541-external-credential-providers)
+- [external kubelet credential providers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2133-kubelet-credential-providers)
+- [New features in Secrets Store CSI driver](https://secrets-store-csi-driver.sigs.k8s.io/introduction.html#features)
+- [Pod Security Policy Replacement](https://github.com/kubernetes/enhancements/issues/2579)
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+- Stable
+    - BoundServiceAccountToken (v1.22)
+    - Certificates API (v1.19)
+    - TokenRequest (v1.20)
+    - TokenRequestProjection (v1.20)
+    - RootCAConfigMap (v1.21)
+    - ServiceAccountIssuerDiscovery (v1.21)
+    - client-go auth plugins (v1.22)
+- Beta
+    - CSIServiceAccountToken (v1.21)
+- Alpha
+    - Hierarchical Namespace Controller
+- Road to alpha
+    - Pod Security Policy Replacement
+
+**What areas and/or subprojects does the group need the most help with?**
+
+- Audit Logging
+    - https://github.com/kubernetes/kubernetes/issues/101597
+    - https://github.com/kubernetes/kubernetes/issues/84571
+    - https://github.com/kubernetes/kubernetes/issues/82295
+    - https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Faudit+
+- Testing
+    - https://github.com/kubernetes/enhancements/issues/541#issuecomment-799372909
+- KMS
+    - [KMS-Plugin: Areas for improvement](https://docs.google.com/document/d/1-WHXX_Dh_MNcJb2QJxF0gOAvLjh0fAnc3QrylWdMZJA/edit)
+
+**What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?**
+
+- Based on devstats [Issue Velocity / Issues age by SIG and repository groups](https://k8s.devstats.cncf.io/d/15/issues-age-by-sig-and-repository-groups?orgId=1&var-period=d7&var-repogroup_name=Kubernetes&var-sig_name=auth&var-kind_name=All&var-prio_name=All) at the time of writing this report, median time to close sig auth issues in Kubernetes/Kubernetes created in the last 7 days is roughly 24 hours.  
```

Comment:
This metric seems very suspect... probably because very few of the issues that aren't closed in 24 hours have been closed at all 😭

Reply:
Yeah this may not be a useful metric.

Reply:
Removed